WorkloadManagementSystem/Agent/MultiProcessorSiteDirector.py

```diff
@@ -165,6 +156,32 @@ def submitJobs( self ):
       siteMask = siteName in siteMaskList
       processorTags = []
 
+      # Check the status of the Site
+      result = self.siteClient.getUsableSites(siteName)
+      if not result['OK']:
+        self.log.error("Can not get the status of site %s: %s" %
+                       (siteName, result['Message']))
+        continue
+      if not result['Value'] or siteName not in result['Value']:
```

Comment:
`if siteName not in result.get('Value', [])` ?

Reply:
OK.